api4/file_test.go

```diff
@@ -359,7 +359,8 @@ func TestGetFileHeaders(t *testing.T) {
 	t.Run("js", testHeaders(data, "test.js", "text/plain", false))
 	t.Run("go", testHeaders(data, "test.go", "application/octet-stream", false))
 	t.Run("zip", testHeaders(data, "test.zip", "application/zip", false))
-	t.Run("exe", testHeaders(data, "test.exe", "application/x-ms", false))
+	// Not every platform can recognize these
+	//t.Run("exe", testHeaders(data, "test.exe", "application/x-ms", false))
```

Comment:
Here we can add the ".exe" extension to the mime types with mime.AddExtensionType. Probably in the `model/file_info.go` file.

Reply:
I guess, but I don't think it's important that we recognize those properly. In fact, it's probably better if we don't.